qt-models/diveimportedmodel.h

```diff
@@ -4,8 +4,9 @@
 #include <QAbstractTableModel>
```

Comment:
The subject line of the commit has a typo: `DiveImporedModel` should be `DiveImportedModel`
And the description in the commit message has two verbs "fill reload"